app/main.py

```diff
@@ -15,5 +15,24 @@
     "Sergio": 3,
 }
 collection_of_coins = {1, 2, 25}
-
-# write your code here
+# write code below this line
+list_of_all_variable = [
+    lucky_number,
+    pi,
+    one_is_a_prime_number,
+    name,
+    my_favourite_films,
+    profile_info,
+    marks,
+    collection_of_coins
+]
+sorted_variables = {}
+mutable_variable_list = []
+immutable_variable_list = []
+for element in list_of_all_variable:
+    if isinstance(element, (int, str, bool, float, tuple)):
+        immutable_variable_list.append(element)
+    else:
+        mutable_variable_list.append(element)
+sorted_variables["mutable"] = mutable_variable_list
+sorted_variables["immutable"] = immutable_variable_list
```

Comment:
you can manually create dictionary making it clearer without the need for extra code

Reply:
if I understand correctly, I should create a dictionary with two keys, add all the values ​​in them and create a check for the values ​​and delete them accordingly.

Reply:
You are correct. Instead of defining variables separately and creating additional code, you can simply define a dictionary with these variables explicitly,  as they are already defined and in our program we don't expect another code